stdlib/SparseArrays/test/sparse.jl

```diff
@@ -368,10 +374,6 @@ end
     @test_throws DimensionMismatch dot(sprand(5,5,0.2),sprand(5,6,0.2))
 end
 
-const BASE_TEST_PATH = joinpath(Sys.BINDIR, "..", "share", "julia", "test")
```

Comment:
Why are you removing the `Quaternion` tests?

Reply:
that was not my intention, sorry - (I did a git fetch upstream before I created the new branch - will be re-inserted.